docs/contracts.rst

```diff
@@ -657,9 +657,9 @@ externally visible functions differ by their Solidity types but not by their ext
 
 ::
 
-    // This will not compile
     pragma solidity ^0.4.16;
 
+    // This will not compile
```

Comment:
This has to be above the pragma to disable compilation of it.

Reply:
I changed the script to accept it anywhere: https://github.com/ethereum/solidity/pull/4788/files#diff-a8a5760144e03e410389fb0afd20e8c8R190